src/FactoryMuffin.php

```diff
@@ -263,7 +263,16 @@ public function instance($name, array $attr = [])
     protected function generate($model, array $attr = [])
     {
         foreach ($attr as $key => $kind) {
-            $model->$key = $this->generatorFactory->generate($kind, $model, $this);
+            $value = $this->generatorFactory->generate($kind, $model, $this);
+
+            $setter = 'set'.$this->camelize($key);
```

Comment:
Good catch. I will fix it, but it won't make a difference since PHP methods are case-insensitive.